src/assets/scripts/client/constants/gameOptionConstants.js

```diff
@@ -34,6 +35,16 @@ export const GAME_OPTION_VALUES = [
         ]
     },
     {
+        name: GAME_OPTION_NAMES.PTL_LENGTH,
+        defaultValue: '1',
+        description: 'Projected Track Line (PTL) Length',
```

Comment:
This wraps because it is a bit too long. Since the whole `Projected Track Line` won't fit, I think we might as well just settle for the abbreviation, despite the downside that many people will not know what a "PTL" is. Though I think they'd probably change the value of the setting and figure it out. Maybe we can do `PTL Length (minutes)` here.

![image](https://cloud.githubusercontent.com/assets/5103735/26664650/21e6cec0-4661-11e7-9136-469d1f563a1d.png)